.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,93 @@
+name: "Bug report"
+description: "Create a report to help us improve TypeScript"
+title: "[BUG] <description>"
+body:
+  - type: markdown
+    attributes:
+      value: Please fill in each section completely. Thank you!
+  - type: input
+    id: search_terms
+    attributes:
+      label: "🔎 Search Terms"
+      description:
+        "What search terms did you use when trying to find an existing bug report?
+
+        List them here so people in the future can find this one more easily."
+    validations:
+      required: true
+  - type: textarea
+    id: version_info
+    attributes:
+      label: "🕗 Version & Regression Information"
+      description: "
+        When did you start seeing this bug occur?
+
+        \"Bugs\" that have existed in TS for a long time are very likely to be FAQs; refer to
+        https://github.com/Microsoft/TypeScript/wiki/FAQ#common-bugs-that-arent-bugs
+
+
+        If possible, please try testing the nightly version of TS to see if it's already been fixed. For npm: `typescript@next`
+
+        This is also the 'Nightly' version in the playground: http://www.typescriptlang.org/play/?ts=Nightly
+  
+
+        Note: The TypeScript Playground can be used to try older versions of TypeScript.
+
+        Please keep and fill in the line that best applies.
+        "
+      value: |
+        - This is a crash
+        - This changed between versions ______ and _______
+        - This is the behavior in every version I tried, and I reviewed the FAQ for entries about _________
+        - I was unable to test this on prior versions because _______
+      render: Markdown
+    validations:
+      required: true
+  - type: input
```

Comment:
I'm curious how this will end up rendering; these links are notoriously long, so hopefully GH can do better here in the final output.

Reply:
Checking https://github.com/jakebailey/ts-issue-template-testing/issues/1, it is small in the form but gets pasted as a huge link here.

I wonder if GH has some special thing that can make links, but I doubt it.